src/utils/get.type.ts

```diff
@@ -0,0 +1,10 @@
+import { supabase } from '$lib/auth';
+
+export async function getProfiles() {
+	const { data } = await supabase.from('Profile').select('*');
+	// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
You shouldn't have the right to write `data!.[0]`.
Remove the `eslint-disable` line and fix `data![0]`.